code/modules/cm_preds/yaut_bracers.dm

```diff
@@ -1260,21 +1260,24 @@
 		to_chat(caller, SPAN_DANGER("You cannot translate (muted)."))
 		return
 
-	caller.create_typing_indicator()
+	var/list/heard = get_mobs_in_view(7, caller)
+	for(var/mob/M in heard)
+		if(M.ear_deaf)
+			heard -= M
+
+	var/image/translator_bubble = image('icons/mob/effects/talk.dmi', src, "pred_translator", TYPING_LAYER)
+	caller.show_speech_bubble(heard, looping_bubble = TRUE, animated = FALSE, speech_bubble = translator_bubble)
 	var/msg = sanitize(input(caller, "Your bracer beeps and waits patiently for you to input your message.", "Translator", "") as text)
```

Comment:
might wanna just convert this to tg say as well, one line change that makes the hud look better